website/project/model.py

```diff
@@ -2749,6 +2749,8 @@ def embargo_registration(self, user, end_date, for_existing_registration=False):
         )
         # Embargo record needs to be saved to ensure the forward reference Node->Embargo
         self.embargo = embargo
+        if self.is_public:
+            self.set_privacy('private', Auth(user))
```

Comment:
They were my concern. Right now they are just stubs that `pass` upon execution but that might not be the case later -- hence the desire for being able to do something along the lines of set_privacy(log=False)